applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -365,120 +428,129 @@ endpoint_format_from(Endpoint, CIDNum, DefaultRealm, CCVs) ->
-                                ], JObj)
-    }.
-
 -spec bridge_from_uri(api_binary(), wh_json:object()) ->
                              api_binary().
-bridge_from_uri(CIDNum, JObj) ->
+bridge_from_uri(Number, JObj) ->
     Realm = wh_json:get_first_defined([<<"From-URI-Realm">>
                                        ,<<"Account-Realm">>
                                       ], JObj),
-    case (whapps_config:get_is_true(?APP_NAME, <<"format_from_uri">>, 'false')
+    case (whapps_config:get_is_true(?SS_CONFIG_CAT, <<"format_from_uri">>, 'false')
           orelse wh_json:is_true(<<"Format-From-URI">>, JObj)
          )
-        andalso (is_binary(CIDNum) andalso is_binary(Realm))
+        andalso (is_binary(Number) andalso is_binary(Realm))
     of
         'false' -> 'undefined';
         'true' ->
-            FromURI = <<"sip:", CIDNum/binary, "@", Realm/binary>>,
+            FromURI = <<"sip:", Number/binary, "@", Realm/binary>>,
             lager:debug("setting bridge from-uri to ~s", [FromURI]),
             FromURI
     end.
 
--spec maybe_emergency_cid_number(wh_json:object()) ->
-                                        api_binary().
-maybe_emergency_cid_number(JObj) ->
-    %% NOTE: if this request had a hunt-account-id then we
-    %%   are assuming it was for a local resource (at the
-    %%   time of this commit offnet DB is still in use)
-    case wh_json:get_value(<<"Hunt-Account-ID">>, JObj) of
-        'undefined' -> emergency_cid_number(JObj);
-        _Else ->
-            wh_json:get_first_defined([<<"Emergency-Caller-ID-Number">>
-                                       ,<<"Outbound-Caller-ID-Number">>
-                                      ], JObj)
+-spec bridge_outbound_cid_name(wh_json:object()) -> api_binary().
+bridge_outbound_cid_name(JObj) ->
+    wh_json:get_first_defined(
+      [<<"Outbound-Caller-ID-Name">>
+       ,<<"Emergency-Caller-ID-Name">>
+      ], JObj
+     ).
+
+-spec bridge_outbound_cid_number(wh_json:object()) -> api_binary().
+bridge_outbound_cid_number(JObj) ->
+    wh_json:get_first_defined(
+      [<<"Outbound-Caller-ID-Number">>
+       ,<<"Emergency-Caller-ID-Number">>
+      ], JObj
+     ).
+
+-spec bridge_emergency_cid_name(wh_json:object()) -> api_binary().
+bridge_emergency_cid_name(JObj) ->
+    wh_json:get_first_defined(
+      [<<"Emergency-Caller-ID-Name">>
+       ,<<"Outbound-Caller-ID-Name">>
+      ], JObj
+     ).
+
+-spec bridge_emergency_cid_number(wh_json:object()) -> api_binary().
+bridge_emergency_cid_number(JObj) ->
+    wh_json:get_first_defined(
+      [<<"Emergency-Caller-ID-Number">>
+       ,<<"Outbound-Caller-ID-Number">>
+      ], JObj
+     ).
+
+-spec find_emergency_number(wh_json:object()) -> api_binary().
+find_emergency_number(JObj) ->
+    case whapps_config:get_is_true(
+           ?SS_CONFIG_CAT
+           ,<<"ensure_valid_emergency_cid">>
+           ,'false'
+          )
+    of
+        'false' ->
```

Comment:
I usually put single-line clauses above multi-line personally when it doesn't affect order of evaluation